src/main/java/nl/unimaas/bigcat/wikipathways/nanopubs/PubMed.java

```diff
@@ -0,0 +1,96 @@
+package nl.unimaas.bigcat.wikipathways.nanopubs;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class PubMed {
+	   public static Date getPublicationDate(String pubmedId) throws IOException, ParseException {
+		   if(isNumeric(pubmedId))
+		   {
+			    String postUrl = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&id="+pubmedId+"&retmode=json"; //post url

```

Comment:
that is going to be a lot of webservice calls :(

Reply:
I resolved the issue by creating the pubmed json data file